resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/repository/AbstractReactiveRepositoryRedisAdapter.java

```diff
@@ -48,12 +52,22 @@
  */
 public abstract class AbstractReactiveRepositoryRedisAdapter<V, K> implements ReactiveCrudRepository<V, K> {
     public static final String KEY_SEP = "::";
+    private static final String ID_KEYSPACE = "id";
+    private static final String PARENT_KEYSPACE = "parent";
+    private static final String FORMAT_KEYSPACE = "format";
+    private static final String TYPE_KEYSPACE = "type";
```

Comment:
Each of these should be within resource, serialisedFormat and type classes respectively

Reply:
Moved to the relevant sub classes